api/src/certification/evaluation/domain/models/CalibratedChallenge.js

```diff
@@ -0,0 +1,35 @@
+export const Accessibility = Object.freeze({
+  RAS: 'RAS',
+  OK: 'OK',
+});
+
+/**
+ * Traduction: Épreuve calibrée
+ */
+export class CalibratedChallenge {
+  /**
+   * Constructeur d'épreuve calibrée
+   *
+   * @param id
+   * @param discriminant
+   * @param difficulty
+   * @param blindnessCompatibility
+   * @param colorBlindnessCompatibility
+   * @param skill
+   */
+  constructor({ id, discriminant, difficulty, blindnessCompatibility, colorBlindnessCompatibility, skill } = {}) {
```

Comment:
Attention, le `{}` indique que basiquement tous les paramètres sont optionnels, je crois comprendre en lisant la PR que ce n'est pas le cas et que beaucoup (si pas tous les champs) sont mandatory.

Pour éviter un objet anémique il faut que le model soit un moyen de comprendre les aspects métier de ce qu'il représente. Les deux modèles créés dans cette PR sont un peu léger sur ce point, en particulier au vu de leur rôle central pour le fonctionnel de Certif 😅 

```suggestion
  constructor({ id, discriminant, difficulty, blindnessCompatibility, colorBlindnessCompatibility, skill }) {
```